templates/components/streamfield/blocks/image_block.html

```diff
@@ -1,5 +1,12 @@
 {% verbatim %}
-<div class="">
-    {{ self }}
+{% load wagtailimages_tags %}
+
+<div class="pb-10 md:pb-20 flex justify-center">
+  <div class="mx-auto">
+    {% image value.image width-640 class="mx-auto" %}
+    {% if value.caption %}
+      <p class="text-xs text-strong mt-4">{{ value.caption }}</p>
```

Comment:
Seems that we need to recompile the CSS for `text-xs` to take effect, but the result seems too tiny:

<img width="1210" alt="image" src="https://github.com/user-attachments/assets/08eb4fcb-5179-452c-8bae-a1b18acc7a67" />

Also, I just realised `text-strong` is not a valid Tailwind class, so it does nothing. I think we can just remove both classes.

```suggestion
      <p class="mt-4">{{ value.caption }}</p>
```

<img width="1210" alt="image" src="https://github.com/user-attachments/assets/4c71fdc0-dfa1-441d-b167-281851c77e74" />